src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -155,3 +260,146 @@ def asset_download(**args):
     Downloads input manifest of previously submitted job.
     """
     click.echo("download complete")
+
+
+def read_local_manifest(manifest: str) -> BaseAssetManifest:
+    """
+    Read manifets specified by filepath to manifest folder, returns BaseAssetManifest Object
+    """
+    for filename in os.listdir(manifest):
+        if filename.endswith("_input"):
+            # decode text file string into BaseAssetManifest
+            filepath = os.path.join(manifest, filename)
+            with open(filepath, "r") as input_file:
+                manifest_data_str = input_file.read()
+                asset_manifest = decode_manifest(manifest_data_str)
+
+                return asset_manifest
+
+    raise NonValidInputError(
+        f"Specified manifests directory {manifest} does not contain manifest file. "
+    )
+
+
+def clear_S3_mapping(manifest: str):
+    """
+    Clears manifest_s3_mapping file contents if it previously exists.
+    """
+    for filename in os.listdir(manifest):
+        if filename.endswith("manifest_s3_mapping"):
+            # if S3 mapping already exists, clear contents
+            filepath = os.path.join(manifest, filename)
+            with open(filepath, "w") as _:
+                pass
+    return
+
+
+def get_directory_changes():
+    """
+    TODO: Gets the file paths in specified directory if the contents of file have changed compared to a specified snapshot of the respective directory.
+    """
+
+    # root_path = asset_root_manifest.root_path
+
+    # input_paths = []
+    # for root, dirs, files in os.walk(root_path):
+    #     if os.path.samefile(root, manifest):
+    #         dirs[:] = []
+    #         continue
+    #     for filename in files:
+    #         file_path = os.path.join(root, filename)
+    #         input_paths.append(file_path)
+
+    # return input_paths
+
+
+def get_manifest_changes(
```

Comment:
nit: instead of the word `get`, could we rename this to `diff_manifest`? It's a bit clearer that way. Right now, the "get" kind of makes it seem like a getter at first glance, when it is really a diff function